src/commands/create-rab-bundle.ts

```diff
@@ -5,12 +5,19 @@
 
 import * as vscode from 'vscode';
 
-import { exportRABBundle } from '../workspace-manager';
+import { firstValueFrom } from 'rxjs';
 import { log } from '../logger';
+import { fs } from '../utils';
 import { showErrorMessage } from '../utils/ui-utils';
+import { exportRABBundle } from '../workspace-manager';
 
 
 async function callback(file: vscode.Uri): Promise<any> {
+  const isSaved = await firstValueFrom(fs.confirmSaveFile(file, true));
```

Comment:
saved file check is already done in bundle validation method.

Reply:
The function creates the experience to request user to select the action:
* Save and proceed
* Cancel

If the user clicks `Save and proceed,` the downstream action will continue. If not, nothing will happen.

It may offer better UX other than just throwing an error and hitting user correct it.

And since this is a pre-check not just for creating bundles but also for Adding Requests from Postman or Openapi Doc, I guess it's good to offer a consistent experience.